src/app/components/Figure/Copyright/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { node } from 'prop-types';
+import styled from 'styled-components';
+import {
+  FF_NEWS_SANS_REG,
+  C_WHITE,
+  C_EBON,
+  GEL_SPACING,
+} from '../../../lib/constants/styles';
+import { T_MINION } from '../../../lib/constants/typography';
+
+// const copyrightSymbolPrefix = '&#169; ';
+const copyrightSymbolPrefix = '© ';
```

Comment:
Reviewer highlight point, should this be the encoded value or just the symbol

Reply:
Probably the encoded value. Is there a reason to not?

Reply:
https://github.com/bbc/simorgh/pull/615/files#r215583168